test/jdk/javax/net/ssl/DTLS/PacketLossRetransmission.java

```diff
@@ -87,6 +88,9 @@ public static void main(String[] args) throws Exception {
         handshakeType = Byte.parseByte(args[1]);
 
         PacketLossRetransmission testCase = new PacketLossRetransmission();
+        if (args.length >= 4 && "shortTimeOut".equals(args[3])) {
```

Comment:
Update this to `equalsIgnoreCase`. 

How do you decide which test to use `shortTimeOut` on? Please add some verbiage as a guidance. Side note - how is `args[2]` used in this test?